alpaca_trade_api/polygon/rest.py

```diff
@@ -134,21 +135,19 @@ def historic_agg_v2(self, symbol, multiplier, timespan, _from, to,
         path_template = '/aggs/ticker/{symbol}/range/{multiplier}/' \
                         '{timespan}/{_from}/{to}'
         if isinstance(_from, int):
-            path = path_template.format(symbol=symbol,
+            pass
+        elif isinstance(_from, datetime.datetime):
+            _from = _from.date().isoformat()
+            to = to.date().isoformat()
```

Comment:
this entire code segment should be in a method called: format_date()
which will make sure _from and to are in the right format.

Reply:
Also, I would handle each parameter separately instead of assuming both are in the same type.